### DISCUSSION TITLE
deepin-image-viewer .webp and .jxl support?

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#2930] [Features Request & Ideas | 特性请求 & 头脑风暴]
I'm starting to receive images in those formats and it would be awesome if deepin could handle them.

**Comment by BLumia** [comment]
deepin-image-viewer already support `.webp` since it's supported by Qt.

`.jxl` can be supported by installing a Qt image formats plugin like [kimageformats](https://invent.kde.org/frameworks/kimageformats) or [qt-jpegxl-image-plugin](https://github.com/novomesk/qt-jpegxl-image-plugin), and after that it will be supported :)

**Reply by raspin0** [reply]
Doesn't work :/ I've installed qt-jpegxl-image-plugin and file manager seems to work fine(thumbnail + preview), but image viewer still doesn't work.

**Reply by BLumia** [reply]
I'll check if it still doesn't work later. btw, which distro are you using now?

**Reply by raspin0** [reply]
Deepin 20.4
